docs/using-with-next.md

```diff
@@ -99,37 +99,31 @@ const AdminApp = () => (
 );
 
 export default AdminApp;
+
 ```
 
-Then we can create the `components/Admin.tsx` by using the same code as the [React Admin Next.js](https://marmelab.com/react-admin/NextJs.html) integration tutorial.
+**Tip:** You can use the  [`recordrepresentation`](https://marmelab.com/react-admin/Resource.html#recordrepresentation) with resources to configure how the record will be rendered by Shadcn Admin Kit.
 
-```tsx
-// components/Admin.tsx
+## Create the Admin Page
+
+Once the <AdminApp /> has been configured, we can create the admin page at `app/admin/page.tsx` to dynamically import our admin component and render it with SSR disabled (since it's a SPA).
 
+```tsx
+// app/admin/page.tsx
 "use client";
+
 import dynamic from "next/dynamic";
 
 const Admin = dynamic(() => import("./AdminApp"), {
   ssr: false, // Required to avoid react-router related errors
 });
 
-export default Admin;
-```
-
-## Create the Admin Page
-
-Once the components have been configured, you can create the admin page at `app/admin/page.tsx`:
-
-```tsx
-// app/admin/page.tsx
-
-import Admin from "@/components/Admin";
-
 export default function Page() {
   return <Admin />;
 }
+
 ```
 
 Then, run `npm run dev` and go to `http://localhost:3000/admin` to access your admin app!
 
-![](./images/shadcn-admin-kit-next/demo.png)
+![A Demo Admin Build with Shadcn Admin Kit](./images/shadcn-admin-kit-next/demo.jpg)
```

Comment:
the screenshot is still huge, can't you take it from a smaller window? :thinking: